wiki/database.md

```diff
@@ -29,20 +29,116 @@ The structure of the DB is the following:
 
 ### Drinks
 
-**OUTDATED**
+A drink is represented by the following JSON: 
 
 ```JSON
 {
-    "drink_name": "Expresso",
-    "drink_price": 4.0,
-    "drink_description": "An expresso coffee",
+    "drink_name": "Name of the coffee recipe or Custom Drink for custom made drinks",
+    "drink_price": "4.0 // nu cred ca ne trebuie",
```

Comment:
Am putea sa lasam, si 0 inseamna ca e free (de ex poti sa ai cafea gratuita la birou, sau un automat de cafea cu plata in hol.

Reply:
Da, cum credeti si voi.